go/private/go_repository.bzl

```diff
@@ -107,3 +107,6 @@ buildifier_repository_only_for_internal_use = repository_rule(
     implementation = _buildifier_repository_impl,
     attrs = _go_repository_attrs,
 )
+
+def flagFromBuildTags(tags):
```

Comment:
I don't think a function is needed to abstract a simple .join -- just do the join inline.